src/stories/MessageArea.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import MessageArea from "../components/MessageArea";
+import "../index.css";
+
+const meta: Meta<typeof MessageArea> = {
+  title: "Components/MessageArea",
+  component: MessageArea,
+};
+
+export default meta;
+type Story = StoryObj<typeof MessageArea>;
+
+export const Default: Story = {
+  render: () => (
+    <MessageArea
+      initialMessages={[
```

Comment:
I like that you have `initialMessages` defined here, but let's also make it a prop so that users can adjust the object and see how the component works with different values.